libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/route/NavigationRouteEx.kt

```diff
@@ -21,12 +22,31 @@ import com.mapbox.navigation.utils.internal.logE
 import com.mapbox.navigator.Navigator
 import com.mapbox.navigator.RouteInterface
 import com.mapbox.navigator.RouterOrigin
-import org.jetbrains.annotations.TestOnly
 
 private const val ROUTE_REFRESH_LOG_CATEGORY = "RouteRefresh"
 
 val NavigationRoute.routerOrigin: RouterOrigin get() = nativeRoute.routerOrigin
 
+/**
+ * Convert [DirectionsRoute] to [NavigationRoute] without
+ * looking the corresponding object up in [RouteCompatibilityCache].
```

Comment:
What's wrong with using cache in tests?

Reply:
When you mock native parser, it won't be invoked if there is such a route in the cache. And some tests rely on what the native parser would return.

Reply:
Can you please share an example of such test?